transcom/mymove
# B20884-INT-fix prime weight tool tip

## Agility ticket
[B20884](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20884)

## Summary
The change was stop a weight tool tip from showing on the move "home" page for the customer for HHG type shipments (i.e. HHG, NTS, NTS-R, Boat, Mobile Home).

### How to test
* Customer:
  - Create a move
  - Add shipments: PPM, HHG, NTS, NTS-R, Boat Haul Away, Boat Tow Away, Mobile Home
  - Go to the move "home" page where all the shipments are listed
  - Verify weight tool tip doesn't show up for any of the shipments (see testing screenshot for example of the tool tip).

## ANDI Screenshots
![image](https://github.com/user-attachments/assets/50488daa-e6b4-42cc-b321-b159f6b4f503)

## Testing Screenshots

Move "home" page:
![image](https://github.com/user-attachments/assets/01728915-e2af-45d9-b56f-2103fa087ea4)

---------------------
Tool tip example:
![image](https://github.com/user-attachments/assets/cfefa719-84aa-4991-b9fd-a23f217b2b5f)

![image](https://github.com/user-attachments/assets/f4be714e-98a2-4351-8c98-c203c32b812f)
